client/app/screen/AssessmentReportScreen.js

```diff
@@ -111,7 +110,10 @@ const AssessmentReportScreen = () => {
                   {course.name}
                 </Dropdown.Item>
               );
-            })}
+            })
+          ) : (
+            <Dropdown.Item>No course yet</Dropdown.Item>
```

Comment:
show no course yet prompt inside dropdown menu instead of show 0 in dropdown menu